article/app/controllers/ArticleController.scala

```diff
@@ -83,14 +79,16 @@ class ArticleController(contentApiClient: ContentApiClient, val controllerCompon
     DotcomponentsDataModel.toJsonString(DotcomponentsDataModel.fromArticle(article, request, blocks))
 
   private def render(path: String, article: ArticlePage, blocks: Blocks)(implicit request: RequestHeader): Future[Result] = {
-    Future {
-      request.getRequestFormat match {
-        case JsonFormat if request.isGuui => common.renderJson(getGuuiJson(article, blocks), article).as("application/json")
-        case JsonFormat => common.renderJson(getJson(article), article)
-        case EmailFormat => common.renderEmail(ArticleEmailHtmlPage.html(article), article)
-        case HtmlFormat => common.renderHtml(ArticleHtmlPage.html(article), article)
-        case AmpFormat => common.renderHtml(views.html.articleAMP(article), article)
-      }
+    val tier = ArticlePicker.getTier(article)
+    val isAmpSupported = article.article.content.shouldAmplify
+    request.getRequestFormat match {
+      case JsonFormat if request.isGuui => Future.successful(common.renderJson(getGuuiJson(article, blocks), article).as("application/json"))
+      case JsonFormat => Future.successful(common.renderJson(getJson(article), article))
+      case EmailFormat => Future.successful(common.renderEmail(ArticleEmailHtmlPage.html(article), article))
+      case HtmlFormat if tier == RemoteRender => remoteRenderer.getArticle(ws, path, article, blocks)
+      case HtmlFormat => Future.successful(common.renderHtml(ArticleHtmlPage.html(article), article))
+      case AmpFormat if isAmpSupported => remoteRenderer.getAMPArticle(ws, path, article, blocks)
+      case AmpFormat => Future.successful(common.renderHtml(ArticleHtmlPage.html(article), article))
```

Comment:
Perhaps we should just 404 here?

Reply:
https://trello.com/c/RcycqQxB/640-should-we-404-non-ampable-articles-or-serve-them-as-html